services/brig/schema/src/V62_users_pending_activation.hs

```diff
@@ -0,0 +1,38 @@
+-- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2020 Wire Swiss GmbH <opensource@wire.com>
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option) any
+-- later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+
+module V62_users_pending_activation (migration) where
+
+import Cassandra.Schema
+import Imports
+import Text.RawString.QQ
+
+migration :: Migration
+migration =
+  Migration 62 "Add users_pending_activation" $
+    schema'
+      [r|
+        CREATE TABLE users_pending_activation
+        (
+          expires_at_day  int
```

Comment:
This is date of the `expires_at` timestamp/
We use `int` for the encoding instead of `date`, because the cassandra lib doesn't seem to implement `date` literals.

Reply:
I just found out that while [cassandra-util](https://hoogle.zinfra.io/file/wire-server/.stack-work/install/x86_64-linux/dd82fa75c77cff497223fd50d01ad955b5a7ef42b25996a8a0e9a15d7718cbc4/8.8.4/doc/cassandra-util-0.16.5/Cassandra.html#t:Value) doesnt provide a `CqlDate` constructor, [cql](https://hoogle.zinfra.io/file/root/.stack/snapshots/x86_64-linux/de08094f0e7de5d808e5aaad6423b6aea588428a561794bbc07b2b7f1c963979/8.8.4/doc/cql-4.0.2/Database-CQL-Protocol.html#t:Value) does. However it's `CqlDate Int32`, so it's not immediately clear how dates are encoded as integers in cassandra.

Reply:
.. I did some investigation `2020-12-01` gets encoded to `-2147465051`

Reply:
Decided not go forward with CqlDate